dashboard/src/main/java/org/openedx/courses/presentation/DashboardGalleryView.kt

```diff
@@ -671,15 +671,21 @@ private fun PrimaryCourseButtons(
     if (!pastAssignments.isNullOrEmpty()) {
         viewsList.add {
             val nearestAssignment = pastAssignments.maxBy { it.date }
-            val title = if (pastAssignments.size == 1) nearestAssignment.title else null
+            val title = if (pastAssignments.size == 1) {
```

Comment:
It will be great if you can add a small doc related to multiple items in primary course card.

Reply:
Added. Please review.